src/components/App.js

```diff
@@ -257,14 +288,16 @@ var App = React.createClass({
         onOpenSettings={this.openSettings}
         onDisconnect={this.disconnect}
         currentChannel={location}
-        username={this.state.user ? this.state.user.name : ""}
+        username={this.state.user ? this.state.user.name : ''}
         requirePassword={this.state.requirePassword}
         theme={this.state.theme}
         left={this.state.leftSidePanel}
         networkName={this.state.networkName}
         joiningToChannel={this.state.joiningToChannel}
       />
-    ) : ""
+    ) : (
```

Comment:
Stylistic nitpicking: Why the `()` and line break (linter again?)? Could this be simple `''`?